src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
```

Comment:
Consider renaming the parameter 'days' to 'numberOfDays' for better clarity and descriptiveness.